_tools/generateFontData.js

```diff
@@ -59,46 +72,58 @@ const writeStylesheet = async (fontData, fontFilePath) => {
 		path.dirname(fontsStylesheetPath),
 		fontFilePath
 	);
-	const stylesheet = buildStylesheet(fontData, fontUrl).toString();
-	return writeFile(fontsStylesheetPath, stylesheet);
+	let stylesheet = buildStylesheet(fontData, fontUrl).toString();
+	stylesheet += "\n\n";
+	return writeFile(fontsStylesheetPath, stylesheet, true);
 };
 
 const writeFontJs = async fontData => {
 	const js = buildFontJs(fontData);
-	return writeFile(fontJsPath, js);
+	return writeFile(fontJsPath, js, true);
 };
 
-const findFirstFontFile = async directory => {
+const findFontFile = async directory => {
 	const fontFiles = (await util.promisify(fs.readdir)(directory)).filter(
 		f => path.extname(f) == ".woff2"
 	);
 
 	assert(
 		fontFiles.length > 0,
-		`No font file found. Place your font in ${path.relative(
+		`No WOFF2 font files found. Place your WOFF2 fonts in ${path.relative(
 			process.cwd(),
 			directory
 		)}.`
 	);
 
-	assert(
-		fontFiles.length == 1,
-		"Multiple font files found. Please specify the path to your font file explicitly."
-	);
+	const paths = fontFiles.map(fontFile => ({
```

Comment:
Nice! You're getting the hang of it ;-) (`map`).

Reply:
✊